README-template.md

```diff
@@ -12,56 +12,52 @@ Please check the [releases](https://github.com/rancher/rke/releases/) page.
 
 ## Requirements
 
-Please review the [OS requirements](https://rancher.com/docs/rke/latest/en/os/) for each node in your Kubernetes cluster.
+Please review the [Requirements](https://rke.docs.rancher.com/os) for each node in your Kubernetes cluster.
 
 ## Getting Started
 
-Please refer to our [RKE docs](https://rancher.com/docs/rke/latest/en/) for information on how to get started!
-For cluster config examples, refer to [RKE cluster.yml examples](https://rancher.com/docs/rke/latest/en/example-yamls/)
+Please refer to our [RKE docs](https://rke.docs.rancher.com/) for information on how to get started!
+For cluster config examples, refer to [RKE cluster.yml examples](https://rke.docs.rancher.com/example-yamls)
 
 ## Installing Rancher HA using rke
 
-Please use [High Availability (HA) Install](https://rancher.com/docs/rancher/v2.x/en/installation/ha/) to install Rancher in a high-availability configuration.
+Please use [Setting up a High-availability RKE Kubernetes Cluster](https://ranchermanager.docs.rancher.com/how-to-guides/new-user-guides/kubernetes-cluster-setup/rke1-for-rancher) to install Rancher in a high-availability configuration.
 
 ## Building
 
 RKE can be built using the `make` command, and will use the scripts in the `scripts` directory as subcommands. The default subcommand is `ci` and will use `scripts/ci`. Cross compiling can be enabled by setting the environment variable `CROSS=1`. The compiled binaries can be found in the `build/bin` directory. Dependencies are managed by Go modules and can be found in [go.mod](https://github.com/rancher/rke/blob/master/go.mod).
 
-RKE now fetches `data.json` from https://github.com/rancher/kontainer-driver-metadata. To fetch data.json and compile it in rke, run 
+Check [codegen/codegen.go](./codegen/codegen.go) to check the default location for fetching `data.json`. You can override the default location as seen in the example below:
```

Comment:
Check ... to check? Maybe use a synonym for one of them like "verify" or something like "read .... to check"